src/gtfs/mod.rs

```diff
@@ -23,11 +23,46 @@ use collection::CollectionWithId;
 use common_format::manage_calendars;
 use gtfs::read::EquipmentList;
 use model::{Collections, Model};
-use objects::Comment;
+use objects;
 use read_utils::add_prefix;
 use std::path::Path;
 use Result;
 
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq)]
+struct Agency {
+    #[serde(rename = "agency_id")]
+    id: Option<String>,
+    #[serde(rename = "agency_name")]
+    name: String,
+    #[serde(rename = "agency_url")]
+    url: String,
+    #[serde(rename = "agency_timezone")]
+    timezone: Option<String>,
+    #[serde(rename = "agency_lang")]
+    lang: Option<String>,
+    #[serde(rename = "agency_phone")]
+    phone: Option<String>,
+    #[serde(rename = "agency_email")]
+    email: Option<String>,
+}
+
+impl From<objects::Network> for Agency {
+    fn from(n: objects::Network) -> Agency {
+        Agency {
+            id: Some(n.id.clone()),
+            name: n.name.clone(),
```

Comment:
All these clones should be useless as you have the ownership of a Network.

Reply:
I have 
```
error[E0507]: cannot move out of borrowed content
  --> src/gtfs/write.rs:31:22
   |
31 |             id: Some(n.id),
   |                      ^ cannot move out of borrowed content
```